group_vars/openstack

```diff
@@ -16,6 +16,7 @@ openstack_auth:
   password: "{{ lookup('env', 'OS_PASSWORD') }}"
   auth_url: "{{ lookup('env', 'OS_AUTH_URL') }}"
 
+cluster_image: d7df64a3-9b2e-4ff2-820f-4481671d1dcf
```

Comment:
Use the name, `centos-7-20190104`- do we really need the UUID instead?

Reply:
Ack.